docs/parameters.yaml

```diff
@@ -172,33 +236,203 @@ root_default: /etc/pelican/oidc-client-secret
 default: $ConfigBase/oidc-client-secret
 components: ["nsregistry"]
 ---
-name: OIDC.DeviceAuthEndpoint
+name: Registry.OIDCDeviceAuthEndpoint
 description: >-
   A URL describing an OIDC Device Auth Endpoint. This is used by the namespace registry to establish OIDC information
   for authenticated registration.
 type: url
 default: none
 components: ["nsregistry"]
 ---
-name: OIDC.TokenEndpoint
+name: Registry.OIDCTokenEndpoint
```

Comment:
As commented elsewhere, let's move the OIDC items outside the registry app... they can actually be top-level instead of `Server` like I suggested earlier if that makes sense.